.github/workflows/main.yml

```diff
@@ -39,16 +39,43 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Test workflow
+    - name: Test workflow (tumor/normal)
       uses: snakemake/snakemake-github-action@v1.24.0
       with:
         directory: .test
         snakefile: workflow/Snakefile
-        args: "--use-conda --show-failed-logs --cores 3 --conda-cleanup-pkgs cache --all-temp"
+        args: "--configfile .test/config-tumor_normal/config.yaml --use-conda --show-failed-logs --cores 3 --conda-cleanup-pkgs cache"
 
     - name: Test report
       uses: snakemake/snakemake-github-action@v1.24.0
       with:
         directory: .test
         snakefile: workflow/Snakefile
-        args: "--report report.zip"
\ No newline at end of file
+        args: "--configfile .test/config-tumor_normal/config.yaml --report report.zip"
+
+    - name: Test workflow (tumor/normal, ffpe)
+      uses: snakemake/snakemake-github-action@v1.24.0
+      with:
+        directory: .test
+        snakefile: workflow/Snakefile
+        args: "--configfile .test/config-tumor_normal_ffpe/config.yaml --use-conda --show-failed-logs --cores 3 --conda-cleanup-pkgs cache --all-temp"
+        stagein: |
+          rm -rf .test/results
+
+    - name: Test workflow (tumor only)
+      uses: snakemake/snakemake-github-action@v1.24.0
+      with:
+        directory: .test
+        snakefile: workflow/Snakefile
+        args: "--configfile .test/config-tumor_only/config.yaml --use-conda --show-failed-logs --cores 3 --conda-cleanup-pkgs cache --all-temp"
+        stagein: |
+          rm -rf .test/results
```

Comment:
Finally, all tests are running.
I guess the `stagein` for removing results from previous tests is not required in case the `--all-temp` parameter of snakemake is set?

Reply:
Yes, but it also doesn't hurt for now. Let's keep it.